### PR TITLE
Fix cargo initialisation

### LINKED ISSUE #17555: Pillboxes do not fire
### Issue Summary
Pillboxes and camo pillboxes do not fire and do not even seem capable of firing as there's no mechanism order them to. Turrets still fire, however.

#### System Information
- **Operating System:** Arch Linux
- **.NET / Mono Version:** Mono 6.4.0.
- **OpenRA Version:** 7622147.
- **Mod:** Red Alert.

#### Additional Information:
Under OpenRA, built from commit 7622147 load this saved game:
[Ascent.zip](https://github.com/OpenRA/OpenRA/files/4021974/Ascent.zip)

- Logs
[logs-20200104.zip](https://github.com/OpenRA/OpenRA/files/4021977/logs-20200104.zip)

- OpenRA Replays
[replay-20200104.zip](https://github.com/OpenRA/OpenRA/files/4021978/replay-20200104.zip)

### PR DESCRIPTION
**Pull request by abcdefg30**
Closes #17555. (See https://github.com/OpenRA/OpenRA/pull/17376#discussion_r357881791)

### CODE REVIEW

**Comment by pchote** [comment]
I suspect that this is still too early. We can't guarantee that the `ConditionManager` has been initialized before `Cargo`, so the condition granting may not work.

**Comment by abcdefg30** [comment]
Moved in a `FrameEndTask` as discussed on IRC.

**Comment by fusion809** [comment]
I can at least confirm that it does, indeed, fix the problem.

File: OpenRA.Mods.Common/Traits/Cargo.cs [hunk 1]
```diff
@@ -192,6 +183,23 @@ void INotifyCreated.Created(Actor self)
 				if (!string.IsNullOrEmpty(Info.LoadedCondition))
 					loadedTokens.Push(conditionManager.GrantCondition(self, Info.LoadedCondition));
 			}
+
+			// Defer notifications until we are certain all traits on the cargo are initialised
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Cargo.cs]: The problem here isn't with the cargo, but with `INotifyPassengerEntered` on the transport. Can we tweak this comment to make this clear?

**Review by pchote — DISMISSED** [review]
LGTM, just one minor comment tweak:

**Comment by abcdefg30** [comment]
Updated.

**Review by pchote — APPROVED** [review]
Still LGTM.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/dfea651401d2f449908e17e7464a94025f59af9f...0baf3f1e8a5fdf362c1935598ccd646374fd236d

**Pull request merged**